test_files/file_comment/export_star.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * @fileoverview fileoverview comment before export.
+ */
+
+export * from './comment_before_var';
```

Comment:
Add some docs (below the export if the comments matter) about what this is testing for.